setup.py

```diff
@@ -40,9 +40,7 @@ def _dbt_duckdb_version():
         "dbt-core~=1.7.0",
         "duckdb>=0.7.0",
     ],
-    extras_require={
-        "glue": ["boto3", "mypy-boto3-glue"],
-    },
+    extras_require={"glue": ["boto3", "mypy-boto3-glue"], "md": ["duckdb>=0.7.0,<=0.9.2"]},
```

Comment:
Do you want me to keep this bit in for now?

Reply:
Yes please! I think will will be super helpful to have `pip install dbt-duckdb[md]` as an option ready to go for people to use before we roll out 0.10 support, and I can open a PR to bump this version once that lands

Reply:
(We also need to keep this line in tox.ini)
```
-e.[md]
```